central/globaldb/metrics/metrics.go

```diff
@@ -177,6 +180,27 @@ var (
 		Name:      "postgres_table_toast_bytes",
 		Help:      "bytes being used by toast for a table",
 	}, []string{"Table"})
+
+	PostgresDBSize = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.CentralSubsystem.String(),
+		Name:      "postgres_db_size",
```

Comment:
Can you add `_bytes` as a suffix to the metric name here? That's a common Prometheus convention.